pdm/models/environment.py

```diff
@@ -143,6 +143,10 @@ def get_finder(
                 self.project.config.get("pypi.ca_certs", certifi.where())
             ),
         )
+        if certfn := self.project.config.get("pypi.client_cert"):
```

Comment:
Walrus operator is py38 and above, which is not allowed according to the `requires-python` range.

Reply:
whoops, got it!